apps/platforms/mac/notes/notes.talon

```diff
@@ -0,0 +1,36 @@
+os: mac
+and app: notes
+-
+new note: key(cmd-n)
+duplicate note: key(cmd-d)
+new folder: key(shift-cmd-n)
+toggle folders: key(alt-cmd-s)
+show main: key(cmd-0)
+show as list: key(cmd-1)
+show as gallery: key(cmd-2)
+toggle attachments: key(cmd-3)
+find all: key(alt-cmd-f)
+print note: key(cmd-p)
+attach file: key(shift-cmd-a)
+create link: key(cmd-k)
+insert table: key(alt-cmd-t)
+apply title: key(shift-cmd-t)
+apply heading: key(shift-cmd-h)
+apply subheading: key(shift-cmd-j)
+apply body: key(shift-cmd-b)
+apply mono: key(shift-cmd-m)
+apply bullet: key(shift-cmd-7)
+apply dash: key(shift-cmd-8)
+apply number: key(shift-cmd-9)
+apply checklist: key(shift-cmd-l)
+increase font: key(cmd-+)
+decrease font: key(cmd--)
+indent: key(cmd-])
+unindent: key(cmd-[)
+line break: key(ctrl-enter)
+mark: key(shift-cmd-u)
+move up: key(ctrl-cmd-up)
+move down: key(ctrl-cmd-down)
+zoom in: key(shift-cmd->)
```

Comment:
I would suggest implementing a few of existing actions for some of these, so the standard global commands will work instead of defining new commands:

```
edit.zoom_in()
edit.zoom_out()
edit.indent_more()
edit.indent_less()
```

zoom in/out are defined in standard.talon, and I think some indent commands are defined in generic_editor.talon

Reply:
You are right, I addressed this in the commit https://github.com/knausj85/knausj_talon/pull/649/commits/e9e4613e002841debd7f0ee66b4019071b3b18cd.